trees/RobustTree.py

```diff
@@ -1,105 +1,15 @@
-        # this is where we will initialize the values we want users to provide
-        self.depth = depth
-        self.time_limit = time_limit,
-        self._lambda = _lambda
-
-    def fit(self, X, y):
-        """A reference implementation of a fitting function.
-
-        Parameters
-        ----------
-        X : {array-like, sparse matrix}, shape (n_samples, n_features)
-            The training input samples.
-        y : array-like, shape (n_samples,) or (n_samples, n_outputs)
-            The target values (class labels in classification, real numbers in
-            regression).
-
-        Returns
-        -------
-        self : object
-            Returns self.
-        """
-        X, y = check_X_y(X, y, accept_sparse=True)
-        self.is_fitted_ = True
-
-        # Instantiate tree object here
-        # tree = Tree(self.depth)
-
-        # Code for setting up and running the MIP goes here.
-        # Note that we are taking X and y as array-like objects
-        # primal = FlowOCT(data_train, label, tree, _lambda, time_limit, mode)
-        # primal.create_primal_problem()
-        # primal.model.update()
-        # primal.model.optimize()
-        # end_time = time.time()
-        # solving_time or other potential parameters of interest can be stored within the class: self.solving_time
-        # solving_time = end_time - start_time
-
-        # Here we will want to store these values and any other variables needed for making predictions later
-        # b_value = primal.model.getAttr("X", primal.b)
-        # beta_value = primal.model.getAttr("X", primal.beta)
-        # p_value = primal.model.getAttr("X", primal.p)
-
-        # `fit` should always return `self`
-        return self
-
-    def predict(self, X):
-        """ A reference implementation of a predicting function.
-
-        Parameters
-        ----------
-        X : {array-like, sparse matrix}, shape (n_samples, n_features)
-            The training input samples.
-
-        Returns
-        -------
-        y : ndarray, shape (n_samples,)
-            Returns an array of ones.
-        """
-        X = check_array(X, accept_sparse=True)
-        check_is_fitted(self, 'is_fitted_')
-
-        # Here we would get the predicted values using the `get_predicted_value` function
-        # https://github.com/pashew94/StrongTree/blob/4541fe5b556d15bcd2814b76a9075b943508fb83/Code/StrongTree/utils.py#L77
-
-        # users can either calculate accuracy/mse themselves or we can expose a method based on sklearn.metrics.accuracy_score or some other metric
-
-        prediction = None
-        return prediction
 
+# Include Necessary imports in the same folder
+from trees.utils.Tree import Tree
+from trees.utils.RobustOCT import RobustOCT
+from trees.utils.RobustTreeUtils import mycallback, check_integer, check_same_as_X
+import time
 
 class RobustTreeClassifier(ClassifierMixin, BaseEstimator):
     """ An example classifier which implements a 1-NN algorithm.
```

Comment:
We need to add a short description of the class here because it will be used to create the API documentation